src/odemis/acq/path.py

```diff
@@ -259,6 +260,28 @@
 ALIGN_MODES = {'mirror-align', 'chamber-view', 'fiber-align', 'streak-align', 'spec-focus', 'spec-fiber-focus', 'streak-focus'}
 
 
+class OneTaskExecutor(ThreadPoolExecutor):
+    """
+    Only one task at a time is executed. If several tasks are queued, only the
+    last one is executed.
+    """
+
+    def __init__(self, thread_name_prefix=''):
+        super(OneTaskExecutor, self).__init__(max_workers=1, thread_name_prefix=thread_name_prefix)
+
+    def submit(self, fn, *args, **kwargs):
```

Comment:
what is param fn? doc string?i

Reply:
This is overriding the parent method. So, same signature. That method actually has also no docstring, because it also overrides the method from its parents, but eventually you can find the right docstring. Typically you don't want to duplicate the docstring when overriding methods (because duplication can easily lead to out of date information), unless you change the signature.

Reply:
ok